provider/src/test/kotlin/org/gradle/kotlin/dsl/accessors/ProjectSchemaAccessorsIntegrationTest.kt

```diff
@@ -126,8 +126,8 @@ class ProjectSchemaAccessorsIntegrationTest : AbstractIntegrationTest() {
     @Test
     fun `can access configurations registered by declared plugins via jit accessor`() {
 
-        withFile("settings.gradle", """
-            include "a", "b", "c"
+        withFile("settings.gradle.kts", """
```

Comment:
⭕️  We should leave at least one test checking the combination `settings.gradle` / `build.gradle.kts`.

Reply:
Done